css-box-model/styles.css

```diff
@@ -0,0 +1,9 @@
+button.language {
+  font-size: 24px;
+  background-color: lightgray;
+  margin-left: 12px;
+  margin-right: 12px;
+  padding: 6px, 16px, 6px, 16px;
```

Comment:
Since this is a shorthand property, you should separate these values with just spaces, no commas